CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Changelog
 
 ## vNEXT
-- Migrate IexecAccessors tests Typescript & Hardhat. (#96)
+- Migrate unit test files to Typescript & Hardhat:
+    - IexecAccessors (#96)
+    - IexecCategoryManager (#97)
```

Comment:
Do we want this or set the latest PR on top of the list ?

Reply:
Both are fine for me

Reply:
Done in the last commit. Thanks!